### PR TITLE
Hw02 express

### PR DESCRIPTION
**Pull request by Lvi-Hub**

### CODE REVIEW

File: routes/api/contacts-router copy.js
Comment (ceREAL133) [inline routes/api/contacts-router copy.js:1]: what is this file for? Just delete it

File: schemes/contacts-schemas.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Joi from "joi";
+
+const contactsAddSchema = Joi.object({
+  name: Joi.string().required(),
```
Comment (ceREAL133) [inline schemes/contacts-schemas.js:4]: you should validate email and phone properly

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: routes/api/contacts-router copy.js
Comment (Lvi-Hub) [reply routes/api/contacts-router copy.js:1]: This is only copy of my file. I was create it and then before make decorators I was store it in my project. I would like have it in my project only like nice example.
But if this is problem for you I can delete.

File: schemes/contacts-schemas.js (hunk 1 above)
Comment (Lvi-Hub) [reply schemes/contacts-schemas.js:4]: I think now will be better, I used pattern for validate phone number and also add validation for email.
Thank you for your remarks.

File: routes/api/contacts-router copy.js
Comment (ceREAL133) [reply routes/api/contacts-router copy.js:1]: it's not problem but holding unused code in project is a bad idea. It pollutes your code

File: schemes/contacts-schemas.js (hunk 1 above)
Comment (ceREAL133) [reply schemes/contacts-schemas.js:4]: now it`s better

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**